src/main/java/core/basesyntax/services/ReaderService.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.services;
+
+import java.util.List;
+
+public interface ReaderService {
```

Comment:
```suggestion
public interface FileReadService {
```
Actually, I think in this task is better to have one FileServise interface with two methods (for reading and writing). But as you wish.